src/main/java/core/basesyntax/service/OperationStrategy.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax.service;
+
+public interface OperationStrategy {
+    void getOperation(FruitTransaction.Operation operation);
```

Comment:
getOperation - must return Operation. But this method is void
Think about method name